redhat-developer/developers.redhat.com
# [DEVELOPER-4094] Remove hard-coded production links from Drupal content

This PR attempts to remove any hard-coded production links from content within Drupal. It does this by replacing the FQDN of the current production Drupal instance from the content stored within the Drupal database. 

For example, links are rewritten as follows:

`http://developer-drupal.web.prod.ext.phx2.redhat.com/foo/bar` becomes `/foo/bar`

To test:

- Ensure all tests are reporting as working correctly
- Inspect the Blinkr report attached to this PR. There should be zero incorrect environment warnings in the report.
